apis/apis.yaml

```diff
@@ -11,4 +11,4 @@ uris:
   - /cloudmaster
 tags:
   - key: role
-    value : radio
\ No newline at end of file
+    value : ato
```

Comment:
Maybe we should just remove the tags :) or use generic tags.  I know it's my fault.

Reply:
I removed them from all resources file, but forgot about others. Fixed :-)